db/migrate/20190529101022_add_bank_details_to_tslr_claim.rb

```diff
@@ -0,0 +1,6 @@
+class AddBankDetailsToTslrClaim < ActiveRecord::Migration[5.2]
+  def change
+    add_column :tslr_claims, :account_sort_code, :string, limit: 6
+    add_column :tslr_claims, :account_number, :string, limit: 8
```

Comment:
Can we rename these fields to `bank_account_number` and `bank_sort_code` to be slightly more precise?